docs/developer-docs/latest/plugins/graphql.md

```diff
@@ -891,3 +891,19 @@ mutation {
 :::
 
 Then on each request, send along an `Authorization` header in the form of `{ "Authorization": "Bearer YOUR_JWT_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.
+
+### API tokens
```

Comment:
Should this be at the `##` level so that it doesn't fall under the "Users and Permissions Plugin" section, since API tokens are an alternate option to the plugin?

Although, I wonder if this could be combined or simplified somehow (maybe outside the scope of what you're doing, but a future note).  That is, the authorization header works exactly the same whether it's a JWT or API Token. So we could have a section about "Adding an authorization header" that explains the bearer token is either an API Token, or the JWT returned from a login.

Reply:
You are correct, I fixed the header level. 

On the second point, I agree and it is something we can revisit in the next quarter. We have been discussing a concepts section with @Aurelsicoko and one of the topics that I think we should cover early on is authentication/authorization.